### ISSUE TITLE
[Suggestion] Additional hints for the Starter Block

### ISSUE THREAD

**Issue by ProPeach** [tsunamayo/Starship-EVO#3088]
We've had a few new players mistakenly building spaceships as ground structures recently by placing the starter block on the ground instead of in the air. If adding a method to convert blueprints from ships to structures and back again is unfeasible, I suggest some extra hints to help prevent newer players from making the mistake in the first place 

Something like this, just a little tip to show what kind of construct is suitable for what colour starter block
![image](https://user-images.githubusercontent.com/27857711/96419509-2673a800-11ec-11eb-9a6c-2e111cec72c2.png)
The cute little spaceship/hovercraft icon on the Cores would look pretty too

![image](https://user-images.githubusercontent.com/27857711/96419630-4f943880-11ec-11eb-8eb4-54aa4742638d.png)

**Comment by tsunamayo** [comment]
Yes you are right, also do something to convert ground build to spaceship. Maybe add a legend or a message also, I now have several way to give hint to players... Thx
